pandas/tseries/converter.py

```diff
@@ -216,7 +216,7 @@ def try_parse(values):
                 else:
                     values = [_dt_to_float_ordinal(x) for x in values]
             except Exception:
-                pass
+                values = _dt_to_float_ordinal(values)
```

Comment:
I am not completely sure, but by letting the DatetimeConverter accept for example strings (although string type is not registered), this makes it possible to do `ax.set_xlim('2012-01-01', '2012-01-10')`